text/0152-telemetry-schemas.md

```diff
@@ -0,0 +1,1159 @@
+  It is not a goal. Our goal is narrowly defined to solve the following problem
+  only: to allow OpenTelemetry Semantic Conventions to evolve over time. For
+  that reason this document is concerned with _changes_ to the schema as opposed
+  to the _full state_ of the schema. We do not preclude this though: the schema
+  file format is extensible and in the future may allow defining the full state
+  of the schema, see [Current State in Schema](#current-state-in-schema) in
+  Future Possibilities section).
+
+- We intentionally limit the types of transformations of schemas to the bare
+  minimum that is necessary to handle the most common changes that we believe
+  OpenTelemetry Semantic Conventions will require in the near future. More types
+  of transformations [may be proposed](#other-transformation-types) in the
+  future. This proposal does not attempt to support a comprehensive set of
+  possible transformation types that can handle all possible changes to schemas
+  that we can imagine. That would be too complicated and very likely
+  superfluous. Any new transformation types should be proposed and added in the
+  future to the schema file format when there is an evidence that they are
+  necessary for the evolution of OpenTelemetry.
+
+## Use Cases
+
+This section shows a couple interesting use-cases for the telemetry schemas
+(other uses-cases are also possible, this is not an exhaustive list).
+
+### Full Schema-Aware
+
+Here is an example on a schema-aware observability system:
+
+![Full Schema-Aware](img/0152-otel-schema.png)
+
+Let's have a closer look at what happens with the Telemetry Source and Backend
+pair as the telemetry data is emitted, delivered and stored:
+
+![Source and Backend](img/0152-source-and-backend.png)
+
+In this example the telemetry source produces spans that comply with version
+1.2.0 of OpenTelemetry schema, where "deployment.environment" attribute is used
+to record that the span is coming from production.
+
+The telemetry consumer desires to store the telemetry in version 1.1.0 of
+OpenTelemetry schema. The schema translator compares the schema_url in the
+received span with the desired schema and sees that a version conversion is
+needed. It then applies the change that is described in the schema file and
+renames the attribute from "deployment.environment" to "environment" before
+storing the span.
+
+And here is for example how the schemas can be used to query stored data:
+
+![Query Translate](img/0152-query-translate.png)
+
+### Collector-Assisted Schema Transformation
+
+Here is a somewhat different use case, where the backend is not aware of schemas
+and we rely on OpenTelemetry Collector to translate the telemetry to a schema
+that the backend expects to receive. The "Schema Translate Processor" is
+configured, the target schema_url is specified and all telemetry data that
+passes through the Collector is converted to that target schema:
```

Comment:
Nit: should this only convert telemetry of a compatible schema (or previous schema version?)

I had thought "schema" would be namespaced/split apart in styles (e.g. HTTP), or is it meant to be ONE schema for ALL semantic conventions?

Reply:
There is one OpenTelemetry schema for all semantic conventions.